docs/user_guide/logging.rst

```diff
@@ -0,0 +1,61 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+
+There are also 3 loggers with a special purpose:
+
+=============================== =====================================================================
+Logger name                     Purpose
+=============================== =====================================================================
+:samp:`rebasehelper.traceback`  traceback generated by unhandled exceptions
+:samp:`rebasehelper.summary`    messages forming the rebase summary
+:samp:`rebasehelper.report`     messages forming the textual rebase report (used by text output tool)
+=============================== =====================================================================
+
+
+:program:`rebase-helper` uses :class:`rebasehelper.logger.CustomLogger` logger class which provides extra
+logging levels.
+
+:mod:`rebasehelper.logger` module provides two utility functions to setup default handlers.
+:meth:`rebasehelper.logger.create_stream_handlers` sets up the default console handlers - with level INFO
+and a formatter displaying logging level for :samp:`rebasehelper` logger and with level DEBUG for
+:samp:`rebasehelper.summary` logger. Both of the handlers are instances of
+:class:`rebasehelper.logger.ColorizingStreameHandler` to provide colored output.
+:meth:`rebasehelper.logger.create_file_handlers` sets up the default file handlers for the main :samp:`rebasehelper`
+logger. 3 handlers are created, they differ in the logging level. The used logging levels are DEBUG, VERBOSE and INFO.
+
+
+
+Examples
+--------
+
+.. code-block:: python
+
+    # use a single module and get logs from it
+    import logging
+
+    from rebasehelper.specfile import SpecFile
+
+    logging.basicConfig()
```

Comment:
Sorry, I didn't realize using `basicConfig()` defeats the purpose of the example (as it always uses root logger).
I suggest something like this to keep the code straightforward:
```python
logger = logging.getLogger('rebasehelper.specfile')
logger.addHandler(logging.StreamHandler())
```